README.md

```diff
@@ -37,13 +38,20 @@ Popular Bayes filters are the **Kalman** [1]-[4] and **particle filters** [5]-[7
 
 The aim of this library is to provide _interfaces_ and _implementations_ for new and existing recursive Bayesian filters!
 
+# Installing with conda
+You can install the binaries with conda. All the dependencies will be automatically installed in the conda environment.
+```bash
+conda install -c conda-forge libbayes-filters-lib
+```
 
-# 🎛 Dependencies
+# Building from sources
```

Comment:
Should not we add within this subsection also the part devoted to run tests (as that is only available only after compiling if I am not wrong)?

Thank you